src/load_inputs/load_generators_data.jl

```diff
@@ -501,4 +516,27 @@ function load_vre_stor_data!(inputs_gen::Dict, setup::Dict, path::AbstractString
 		inputs_gen["dfVRE_STOR"] = DataFrame()
 	end
 	summarize_errors(error_strings)
-end
\ No newline at end of file
+end
+
+function get_resources_which_can_be_retired(df::DataFrame)::Set{Int64}
+    if string(:Can_Retire) in names(df)
+        retirable = df[df.Can_Retire.==1, :R_ID]
+
+        invalid_newbuild = df.New_Build.==-1
+        if any(invalid_newbuild)
+            @warn "Deprecated '-1' entries in the 'New_Build' column.
+This previously indicated inability to build or to retire. Entries which previously
+had New_Build=-1 should now have New_Build=0, Can_Retire=0."
+        end
+    else
+        # Backward compatibility.
+        retirable = df[df.New_Build.!=-1, :R_ID]
+        if !isempty(retirable)
+            @info "The generators input file, 'New_Build' column, has some entries
```

Comment:
This info message is not accurate. The "retirable" set which triggers the warning refers to resources with New_Build *not* equal to -1, which is inconsistent with the warning message which complains about resources with New_Build equal to -1. Perhaps we can just throw a more generic warning message about not having the Can_Retire column if this column is not found? And remind users of the correct input format? (What 0/1 means in each column)

Reply:
Plz lmk if this is satisfactory.

Reply:
Putting my Slack comments here for record keeping purposes: I think that we should require the Can_Retire column, and throw the info message anytime it is not present. Otherwise, it will not be possible to change the default Can_Retire value to 0 (which I would assume we would want if this column is optional) without breaking backward compatibility.